### PR TITLE
Srvc 2020 05 31 preview

### PR DESCRIPTION
**Pull request by zolvarga**
# Thank you for helping us improve the Azure IoT Python SDK!

## Need Support

- **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/321918-azure-iot) to help us prioritize
- **Have a technical question?** Ask on [Stack Overflow with tag "azure-iot-hub"](https://stackoverflow.com/questions/tagged/azure-iot-hub)
- **Need Support?** Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response time.  Consider submitting a ticket and get assistance from Microsoft support team
- **Found a bug?** Please help us fix it by thoroughly documenting it and filing an issue on GitHub (See below).

Here's a little checklist of things that will help it make its way to the repository: Note that you don't have to check all the boxes, we can help you with that.
This being said, the more you do, the quicker it'll go through our gated build!
-->

## Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-python/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

## Description of the problem

Please be as precise as possible: what issue you experienced, how often...

## Description of the solution

How you solved the issue and the other things you considered and maybe rejected

### CODE REVIEW

**Comment by cartertinney** [comment]
>  "$id": "1",

I don't understand what this file is

---
Refers to: azure-iot-hub/azure/iot/hub/code-model-v1:2 in 309234f. [](commit_id = 309234f399e0a6e93d76d4adeafd70b7971c504b, deletion_comment = False)

File: azure-iot-hub/azure/iot/hub/iothub_digitaltwin_manager.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        :returns: The return object containing the Digital Twin.
+        """
+
+        return self.protocol.digital_twin.get_digital_twin(digital_twin_id)
+
+    def update_digital_twin(self, digital_twin_id, digital_twin_patch, etag=None):
+        """Update the Digital Twin Component of a given device using a patch object.
+        :param str digital_twin__id: The digital twin Id of the given device.
+        :param list[object]: The json-patch object to update the Digital Twin.
```
Comment (cartertinney) [inline azure-iot-hub/azure/iot/hub/iothub_digitaltwin_manager.py:45]: >object [](start = 20, length = 6)

What's in the list?

File: azure-iot-hub/azure/iot/hub/iothub_digitaltwin_manager.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            digital_twin_id, digital_twin_patch, etag
+        )
+
+    def invoke_component_command(self, digital_twin_id, component_path, command_name, payload):
+
+        """Invoke a command on an component of a particular device and get the result of it.
+        :param str digital_twin__id: The digital twin Id of the given device.
+        :param str component_path: The component's name.
+        :param str command_name: The command's name.
+        :param str The argument of a command.
```
Comment (cartertinney) [inline azure-iot-hub/azure/iot/hub/iothub_digitaltwin_manager.py:64]: >param str [](start = 9, length = 9)

Missing the name of the parameter (payload?)

**Comment by zolvarga** [comment]
>  "$id": "1",

You are right - not needed.  Removing it...

---
In reply to: [621392318](https://github.com/Azure/azure-iot-sdk-python/pull/554#issuecomment-621392318) [](ancestors = 621392318)

---
Refers to: azure-iot-hub/azure/iot/hub/code-model-v1:2 in 309234f. [](commit_id = 309234f399e0a6e93d76d4adeafd70b7971c504b, deletion_comment = False)

**Pull request merged**